docs/ingestion/native-batch.md

```diff
@@ -1064,7 +1064,7 @@ Sample specs:
       "type": "index_parallel",
       "inputSource": {
         "type": "hdfs",
-        "paths": "hdfs://foo/bar/", "hdfs://bar/foo"
+        "paths": "hdfs:/foo/bar/", "hdfs:/bar/foo"
```

Comment:
`hdfs://foo/bar` seems strange to me because it refers the `/bar` path on the host `hdfs://foo` (https://en.wikipedia.org/wiki/Uniform_Resource_Identifier). I don't think it was intentional.

Reply:
I think `hdfs:/foo/bar` may only work if the namenode is configured in hadoop. For general cases, the URI format is `scheme://authority/path` (https://hadoop.apache.org/docs/r2.4.1/hadoop-project-dist/hadoop-common/FileSystemShell.html#Overview)
Do you think it would be clearer to change this to `"hdfs://namenodehost/foo/bar/"` instead?

Reply:
That sounds even better. I will update it.

Reply:
I updated the doc.